src/main/java/com/revature/repositories/PersonRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.revature.repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import com.revature.domain.Person;
+
+@Repository
+public interface PersonRepository extends JpaRepository<Person,Integer> {
+
+	List<Person> findAllByLastName(String lastName);
+	List<Person> findAllByFirstName(String firstName);
+	List<Person> findAllByFirstNameAndLastName(String firstName, String lastName);
+	Person save(Person person);
+	void delete(Person person);
+	
+	//@Query("FROM Person Bob JOIN Eval Frank ON Bob.id=Frank.batch.id WHERE Bob.id=?1")
+	//List<Person> findAllPersonByEvalByBatchById(Integer id);
+	
+	
+	//@Query("SELECT Person.firstName, Person.lastName, Person.id FROM Person INNER JOIN Eval ON Person.id=Eval.Trainee.id WHERE Eval.batch.id=?1")
+	//List<Person> findAllByBatchId(Integer id);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)